plugins/jobs/src/js/components/JobStopRunModal.js

```diff
@@ -0,0 +1,86 @@
+import { Confirm } from "reactjs-components";
+import PropTypes from "prop-types";
+import React from "react";
+import ModalHeading from "#SRC/js/components/modals/ModalHeading";
+
+class JobStopRunModal extends React.Component {
+  getContentHeader(selectedItems, selectedItemsLength) {
+    let headerContent = ` ${selectedItemsLength} Job Runs`;
+    if (selectedItemsLength === 1) {
+      headerContent = "this";
+    }
+
+    return (
+      <ModalHeading key="confirmHeader">
+        {`Are you sure you want to stop ${headerContent}?`}
+      </ModalHeading>
+    );
+  }
+
+  getConfirmTextBody(selectedItems, selectedItemsLength) {
+    let bodyText;
```

Comment:
I still think it's more concise and easier to read and follow :D prettier uses prefix operator notation so `?` and `:` are very prominent.
```js
const bodyText =
  selectedItemsLength === 1
    ? `the job run with id ${selectedItems[0]}`
    : "the selected job runs";
```

Look ma no mutation! :D

Reply:
I agree, but I am still not willing to start improving this UI component out of the scope of refactoring the Store out of it.